src/coreclr/vm/qcallentrypoints.cpp

```diff
@@ -75,6 +75,11 @@
 
 #include "versionresilienthashcode.h"
 
+#ifdef TARGET_BROWSER
+#include "wasm/wasm-entrypoints.h"
```

Comment:
This should just be `entrypoints.h` if we are nesting it in a `wasm` subdirectory.